src/oom/oomd-manager-bus.c

```diff
@@ -38,6 +38,14 @@ static const sd_bus_vtable manager_vtable[] = {
                                  SD_BUS_PARAM(fd),
                                  bus_method_dump_by_fd,
                                  SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_SIGNAL_WITH_NAMES("KilledForMemoryUsed",
+                                 "s",
+                                 SD_BUS_PARAM(name),
+                                 0),
+        SD_BUS_SIGNAL_WITH_NAMES("KilledForMemoryPressure",
+                                 "s",
+                                 SD_BUS_PARAM(name),
+                                 0),
```

Comment:
this really should be the same signal i think, and carry a "reason" string field, or so. which contains "memory-used" or "memory-pressure"

"name" is not a good name, noone knows what that could be

Reply:
Does "cgroup" fit instead of "name"?